### PR TITLE
Navigation and footer

### PR DESCRIPTION
**Pull request by SaurabhChawda**
Nav Footer Pr
I have added Navigation and Footer  Components in trendy-t Ecommerce react App.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,66 +1,59 @@
...
+  --primary-bg-color: rgb(24, 22, 22);
+  --secondary-bg-color: rgb(34, 32, 32);
+  --white-color: white;
+  --black-color: black;
+  --font-size-XX-large: clamp(3rem, 12vw, 10rem);
+  --font-size-X-large: clamp(1.2rem, 2.5vw, 10rem);
+  --font-size-large: clamp(1rem, 1.5vw, 10rem);
+  --font-size-regular: clamp(12px, 2.5vw, 1rem);
+  --font-size-small: clamp(8px, 1vw, 1rem);
+} */
```
Comment (harshitadaswani) [inline src/App.css:43]: remove comments

File: src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./footer.css";
+import Github from "../../assets/image/HomePage-Images/Github.png";
+import Twitter from "../../assets/image/HomePage-Images/Twitter sds.png";
+import LinkedIn from "../../assets/image/HomePage-Images/LinkedIn.png";
```
Comment (harshitadaswani) [inline src/components/Footer/Footer.jsx:5]: you can add the images in public folder to directly use it instead of importing each image as a component

**Review by harshitadaswani — APPROVED** [review]
I liked how you have used `<NavLink />`. Also, please make pull requests short for the reviewer to review them nicely.

**Review by ShreyashMahajan — APPROVED** [review]
Only one suggestion, please try to keep files of smaller size

File: src/App.css (hunk 1 above)
Comment (SaurabhChawda) [reply src/App.css:43]: Okay

File: src/components/Footer/Footer.jsx (hunk 2 above)
Comment (SaurabhChawda) [reply src/components/Footer/Footer.jsx:5]: I will add images folder in public

Comment (SaurabhChawda) [reply src/components/Footer/Footer.jsx:5]: Thankyou.

**Comment by SaurabhChawda** [comment]
> Only one suggestion, please try to keep files of smaller size

Okay, Thankyou

**Pull request merged**